openBCISimulator.js

```diff
@@ -87,13 +92,45 @@ function OpenBCISimulatorFactory() {
         switch (data[0]) {
             case k.OBCIStreamStart:
                 if (!this.stream) this._startStream();
+                this.streaming = true;
                 break;
             case k.OBCIStreamStop:
                 if (this.stream) clearInterval(this.stream); // Stops the stream
+                this.streaming = false;
                 break;
             case k.OBCIMiscSoftReset:
                 if (this.stream) clearInterval(this.stream);
-                this.emit('data', new Buffer('OpenBCI Board Simulator\nPush The World V-0.2\n$$$'));
+                this.streaming = false;
+                this.emit('data', new Buffer('OpenBCI Board Simulator\nPush The World\nFirmware: v2$$$'));
+                break;
+            case k.OBCISDLogForHour1:
```

Comment:
No, they all roll down to the next break which handles all sd card functions. A `case` statement with out a break statement breaks on the next `break`, it does not exit the function.